modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+Partition autobalancer tick interval.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `30s`
+
+---
+
+=== partition_autobalancing_tick_moves_drop_threshold
+
+If the number of scheduled tick moves drops by this ratio, a new tick is scheduled immediately. Valid values are (0, 1]. For example, with a value of 0.2 and 100 scheduled moves in a tick, a new tick is scheduled when the inprogress moves are < 80.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* number
+
+*Default:* `0.2`
+
+---
+
+=== partition_manager_shutdown_watchdog_timeout
+
+A threshold value to detect partitions which shutdown might have been stuck. After this threshold a watchdog in partition manager will log information about partition shutdown not making progress.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `30s`
+
+---
+
+=== pp_sr_smp_max_non_local_requests
+
+Maximum number of x-core requests pending in Panda Proxy and Schema Registry seastar::smp group.  (for more details look at `seastar::smp_service_group` documentation).
+
+*Requires Restart:* Yes
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `null`
+
+---
+
+=== quota_manager_gc_sec
+
+Quota manager GC frequency in milliseconds.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `std::chrono::milliseconds(30000)`
+
+---
+
+=== raft_election_timeout_ms
+
+Election timeout expressed in milliseconds.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `1500ms`
+
+---
+
+=== raft_enable_lw_heartbeat
+
+Enables raft optimization of heartbeats.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `true`
+
+---
+
+=== raft_flush_timer_interval_ms
+
+Interval of checking partition against the `raft_replica_max_pending_flush_bytes`.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `100ms`
+
+---
+
+=== raft_heartbeat_disconnect_failures
+
+After how many failed heartbeats to forcibly close an unresponsive TCP connection.  Set to 0 to disable force disconnection.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `3`
+
+---
+
+=== raft_heartbeat_interval_ms
+
+Milliseconds for raft leader heartbeats.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
```

Comment:
**ENG:** How can you specify a negative number of ms between heartbeats? Makes no sense. Please clarify.